README.md

```diff
@@ -3,6 +3,8 @@
 ![Experimental](https://img.shields.io/badge/experimental-breaking%20changes%20allowed-yellow)
 ![Alpha](https://img.shields.io/badge/alpha-release-red)
 
+> ⚠️ This repositority is a work-in-progress. Tooling and documenation may be incomplete. ⚠️
```

Comment:
Please also mention there's no published SDK and that the SDK on which these components depend will be subject to potentially frequent and large changes

Reply:
Better?

https://github.com/open-feature/python-sdk-contrib/pull/4/files?short_path=b335630#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5